recipe/meta.yaml

```diff
@@ -1,33 +1,32 @@
-{% set version = "4.0.1" %}
+{% set version = "6.0.0" %}
 
 package:
   name: flake8
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/f/flake8/flake8-{{ version }}.tar.gz
-  sha256: 806e034dda44114815e23c16ef92f95c91e4c71100ff52813adf7132a6ad870d
+  sha256: c61007e76655af75e6785a931f452915b371dc48f56efd765247c8fe68f2b181
 
 build:
-  number: 1
-  noarch: python
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - flake8 = flake8.main.cli:main
 
 requirements:
   host:
-    - python
+    - python >=3.8.1
```

Comment:
You may still want to add a `skip: True` for anything below `py38` so that our builders don't try to build it for `py37`.

Reply:
Thanks! Added